modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### 5.9.0
+
```

Comment:
This should not be here sorry @camiloHimura, it should only be under `Unreleased`.
```suggestion
```